test/e2e/specs/specs-playwright/wp-gutenboarding__create.ts

```diff
@@ -0,0 +1,86 @@
+import {
+	setupHooks,
+	DataHelper,
+	LoginFlow,
+	GutenbergEditorPage,
+	SidebarComponent,
+	SiteSelectComponent,
+	GutenboardingFlow,
+	GeneralSettingsPage,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe( DataHelper.createSuiteTitle( 'Gutenboarding: Create' ), function () {
+	const siteTitle = DataHelper.getBlogName();
+
+	let siteURL: string;
+	let gutenboardingFlow: GutenboardingFlow;
+	let page: Page;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	it( 'Log in', async function () {
+		const loginFlow = new LoginFlow( page, 'defaultUser' );
+		await loginFlow.logIn();
+	} );
+
+	it( 'Click on Add Site on Sidebar', async function () {
+		const sidebarComponent = new SidebarComponent( page );
+		await sidebarComponent.switchSite();
+		const siteSelectComponent = new SiteSelectComponent( page );
+		await siteSelectComponent.addSite();
+	} );
+
+	it( 'Enter new site name', async function () {
+		gutenboardingFlow = new GutenboardingFlow( page );
+		await gutenboardingFlow.enterSiteTitle( siteTitle );
+		await gutenboardingFlow.clickButton( 'Continue' );
+	} );
+
+	it( 'Search for and select a WordPress.comdomain name', async function () {
+		await gutenboardingFlow.searchDomain( siteTitle );
+		siteURL = await gutenboardingFlow.selectDomain( { name: siteTitle, tld: '.wordpress.com' } );
+		await gutenboardingFlow.clickButton( 'Continue' );
+	} );
+
+	it( 'Select Vesta as the site design', async function () {
+		await gutenboardingFlow.selectDesign( 'Vesta' );
+	} );
+
+	it( 'Pick the Playfair font pairing', async function () {
+		await gutenboardingFlow.selectFont( 'Playfair' );
+		await gutenboardingFlow.clickButton( 'Continue' );
+	} );
+
+	it( 'Select to add the Plugin feature', async function () {
+		await gutenboardingFlow.selectFeatures( [ 'Plugins' ] );
+		await gutenboardingFlow.clickButton( 'Continue' );
+	} );
+
+	it( 'WordPress.com Business plan is recommended', async function () {
+		await gutenboardingFlow.validateRecommendedPlan( 'Business' );
+	} );
+
+	it( 'Select free plan', async function () {
+		await gutenboardingFlow.selectPlan( 'Free' );
+	} );
+
+	it( 'See the Gutenberg editor', async function () {
+		const gutenbergEditorPage = new GutenbergEditorPage( page );
+		await gutenbergEditorPage.waitUntilLoaded();
+	} );
+
+	it( `Delete created site`, async function () {
+		const settingsURL = DataHelper.getCalypsoURL( `settings/general/${ siteURL }` );
+		await page.goto( settingsURL, { waitUntil: 'load' } );
+		const generalSettingsPage = new GeneralSettingsPage( page );
+		try {
+			await generalSettingsPage.deleteSite( siteURL );
+		} catch ( e ) {
+			console.error( `Deleting ${ siteURL } failed, manual intervention needed.` );
```

Comment:
Out of curiosity: is manual intervention really needed here? I don't think this will disrupt future runs - it will just orphan a test site. That's already something we have a ton of, and I think it makes more sense to try to clean those up with some background job that putting the onus on a test runner (who might be hesitant to then re-run the test)

Reply:
It will orphan the site - that is correct, but the buildup of orphaned site that belongs to the test user is what ended up slowing down the said test user considerably, so I would prefer that anyone investigating and seeing the error message deal with it.

Probably won't happen, though 😆

Reply:
I've adjusted the error message so it doesn't call for manual intervention now.

Reply:
Fixed in 32357f3ed0c917646b87a95c963f9109d720b478.